tests/integration_tests/releases.py

```diff
@@ -0,0 +1,93 @@
+import functools
+import logging
+from typing import Optional
+
+from packaging import version
```

Comment:
Is packaging an indirect dependency? I think we should probably make this an explicit requirement if we're going to explicitly use it.